eixampledb-core/src/main/java/com/eixampledb/core/api/OperationValueTypeException.java

```diff
@@ -0,0 +1,26 @@
+package com.eixampledb.core.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class OperationValueTypeException extends RuntimeException {
```

Comment:
When this exception is thrown Springframework automatically responds with a Bad Request status code.